azext_iot/central/params.py

```diff
@@ -54,7 +54,13 @@ def load_central_arguments(self, _):
             help="The App ID of the IoT Central app you want to manage."
             ' You can find the App ID in the "About" page for your application under the help menu.',
         )
-        context.argument("api_version", arg_type=api_version)
+        context.argument(
+            "api_version",
+            arg_type=api_version,
+            help="This command parameter has been deprecated and will be ignored."
+            "In the future release, we will only support IoT Central APIs from latest GA version."
+            "If any API is not GA yet, we will call latest preview version.",
```

Comment:
will there be a situation where command x has a GA api and a preview api?

Reply:
Good point. But I don't think so. Most commands are calling one API at one time.